user_modules/shared_testcases/testCasesForRAI.lua

```diff
@@ -0,0 +1,169 @@
+local testCasesForRAI = {}
+
+require('atf.util')
+local events = require("events")
+
+--[[@InitHMI_onReady_without_UI_GetCapabilities: replace original InitHMIOnReady from connecttest 
+--! without expect UI.GetCapabilites
+--! @parameters: NO
+--]]
+function testCasesForRAI:InitHMI_onReady_without_UI_GetCapabilities(self)
```

Comment:
@istoimenova 
Could you please explaine why you use method syntax (table:method) for this function and shadowing parameter `self?
May be you can use function syntax (table.function) instead?  
In this case you can use more describtive name for parameter.